jujugui/static/gui/src/app/app.js

```diff
@@ -825,9 +825,10 @@ YUI.add('juju-gui', function(Y) {
     _renderCharmbrowser: function(metadata) {
       var state = this.state;
       var utils = views.utils;
+      var cs = this.get('charmstore');
       React.render(
         <components.Charmbrowser
-          charmstore={this.get('charmstore')}
+          search={cs.search.bind(cs)}
```

Comment:
Just for the sake of clarity it might be nice for this to be passed in as searchCharmstore or charmstoreSearch or whatever makes sense, just to help us know what this method does (it might also prevent conflicts with other props named search as it's a common term).